src/styles/calendar.scss

```diff
@@ -0,0 +1,45 @@
+@use './variables.scss' as *;
+
+.calendar {
+    display: flex;
+    flex-wrap: wrap;
+    gap: 1px;
+    width: calc(#{$calendar-day--size * 7} + 26px); //  26px == 20px padding + 6px gaps
```

Comment:
Ensure that the variable `$calendar-day--size` is defined in the `variables.scss` file. Without this variable, the calculation for the width will not work.